src/swolecore/swolecore.csproj

```diff
@@ -4,4 +4,16 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="google.protobuf" Version="3.14.0" />
+    <PackageReference Include="Grpc.Tools" Version="2.34.0">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Protobuf Include="./messages.proto" />

```

Comment:
we'll probably need a more descriptive name but it can wait until this evolves.

Reply:
yeah i agree, just wasn't sure what the convention is for protobuf. is it generally one file with all the messages or a set of files for each?